packages/waku/src/lib/middleware/rsc/worker-impl.ts

```diff
@@ -208,6 +208,9 @@ const resolveClientEntry = (
   command: 'dev' | 'build' | 'start',
   resolveClientPath: Entries['resolveClientPath'],
 ) => {
+  filePath = filePath.startsWith('file:///')
```

Comment:
Does triple slash mean it's only for posix?

Reply:
This is just a check, won't remove the prefix `/`

Reply:
yeah, but I mean for windows, is it like `file://d/...`?

Reply:
I bet there only have file:///C://

let me try it tomorrow

Reply:
oh i see.